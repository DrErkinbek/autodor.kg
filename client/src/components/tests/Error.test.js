import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import Error from '../layout/404';

it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<Error />, div);
});

it('renders correctly and equal to snapshot', () => {
    const tree = renderer.create(<Error />).toJSON();
  
    expect(tree).toMatchSnapshot();
});