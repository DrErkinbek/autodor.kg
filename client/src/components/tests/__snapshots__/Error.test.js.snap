// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly and equal to snapshot 1`] = `
<div
  className="row"
>
  <div
    className="container"
  >
    <div
      className="notfound-wrapper"
    >
      <h5
        className="notfound-title center blue-text center"
      >
        404
      </h5>
      <p
        className="notfound-tag-1 center center"
      >
        Страница не найдена!!!
      </p>
      <br />
      <div
        className="center"
      >
        <a
          className="waves-effect waves-light btn blue"
          href="/"
        >
          <i
            className="material-icons left"
          >
            reply
          </i>
          Назад в Главную
        </a>
      </div>
    </div>
  </div>
</div>
`;
